docs/source/release/v6.12.0/Framework/Dependencies/New_features/38420.rst

```diff
@@ -0,0 +1 @@
+- Updated Numpy to version 2. `Read about the changes <https://numpy.org/news/#numpy-200-released>`_. **Users should note that Numpy 2 introduces some breaking API changes, see the `Numpy 2 Migration Guide <https://numpy.org/devdocs/numpy_2_0_migration_guide.html>`_ for more details**
```

Comment:
Normally we are specific about minor version support when updating our depenency on numpy.
```suggestion
- Drop support for NumPy version 1. We now build against NumPy v2.0 and support up to v2.1. `Read about the changes <https://numpy.org/news/#numpy-200-released>`_. **Users should note that NumPy 2 introduces some breaking API changes. See the `NumPy 2 Migration Guide <https://numpy.org/devdocs/numpy_2_0_migration_guide.html>`_ for more details**
```